lib/include/robot_interface/robot_interface.h

```diff
@@ -74,6 +74,10 @@ class RobotInterface
     ros::Subscriber            _endpt_sub;
     std::vector<double>       _filt_force;
     double                    force_thres;
+    std::vector<double>      _filt_change;
+    double                  _filt_updated;
```

Comment:
Also, store the time as a ros::Time variable (see below the `time_start` variable for inspiration).